src/docs/community/migration.mdx

```diff
@@ -0,0 +1,242 @@
+import { Divider } from '../../app/components';
+
+<!-- KEYWORDS: developer develop environment guide tutorial -->
+
+# Package Migration Notice
+
+## @brightlayer-ui/angular-auth-workflow
+
+-   `<pxb->` preface changed to `<blui->`.
```

Comment:
I'd make this more like an instruction. For example, "Change all `pxb-` component prefixes to `blui-`". Right now it looks like a changelog rather than an instruction to upgrade.